web-client/integration-tests/journey/docketClerkConsolidatesCaseThatCannotBeConsolidated.js

```diff
@@ -11,7 +11,7 @@ export const docketClerkConsolidatesCaseThatCannotBeConsolidated =
         'Place of trial is not the same',
       ]);
       expect(
-        cerebralTest.getState('caseDetail.consolidatedCases'),
-      ).toBeUndefined();
+        cerebralTest.getState('caseDetail.consolidatedCases').length,
+      ).toEqual(0);
```

Comment:
❓ I am confused by this change. Was this test failing before? I don't see any changes that should have changed the behavior such that we were getting an undefined thing before, and now we're getting an empty array.

Reply:
So the nature of the bug fix was that consolidated cases were disappearing from state after using any of the case functions in the Case Information Tab. Our fix was to, by default, set them again in state in all of the associated sequences. Because of this fix the property consolidatedCases on caseDetail can no longer be undefined. It is an empty array, indicating that there are no associated consolidated cases.